packages/flutterfire_ui/example/test_driver/integration_test.dart

```diff
@@ -0,0 +1,3 @@
+import 'package:integration_test/integration_test_driver.dart';
+
+Future<void> main() => integrationDriver();
```

Comment:
Are the tests of `flutterfire` used? They are not in the CI, right?

Reply:
We don't have integration tests for `flutterfire_ui` if that is your question 😄

Reply:
But there are some tests in `example` folder of the `flutterfire_ui` 🤔 https://github.com/firebase/flutterfire/tree/master/packages/flutterfire_ui/example/test_driver

Reply:
Huh, they aren't part of our CI e2e tests. I think you can leave them for now and we'll integrate them at some other point. Unless @lesnitsky thinks otherwise?

Reply:
Should we add them also? If yes, I can make a new issue and integrate them in a separate PR into our CI 👍